http-endpoint/src/lib.rs

```diff
@@ -44,6 +45,13 @@ pub struct Config {
 
     #[serde(default)]
     pub http: HttpConfig,
+
```

Comment:
I think we should extract that into a dedicated struct, having a field `cors` in the http configuration.